ghost/core/core/server/models/member.js

```diff
@@ -154,11 +154,7 @@ const Member = ghostBookshelf.Model.extend({
             .query((qb) => {
                 // avoids bookshelf adding a `DISTINCT` to the query
                 // we know the result set will already be unique and DISTINCT hurts query performance
-                if (labs.isSet('compExpiring')) {
-                    qb.columns('products.*', 'expiry_at');
```

Comment:
If we're removing `compExpiring`, do we not need to keep `                    qb.columns('products.*', 'expiry_at');`?